prelude/__internal__/Unit.mad

```diff
@@ -0,0 +1 @@
+export data Unit = Unit
```

Comment:
@aboeglin Either get rid of this altogether, or create a ticket to maybe make it a first order constant type with the unique `()` constructor ?